isis/tests/CMakeLists.txt

```diff
@@ -3,13 +3,13 @@ cmake_minimum_required(VERSION 3.10)
 add_dependencies(isis3 isis3)
 
 file(GLOB test_source "${CMAKE_SOURCE_DIR}/tests/*.cpp")
-
+file(GLOB isis_libs "${CMAKE_BINARY_DIR}/lib/lib*")
 
 # Link runISISTests with what we want to test and the GTest and pthread library
 add_executable(runISISTests
                IsisTestMain.cpp
                ${test_source})
 
-target_link_libraries(runISISTests isis3 ${ALLLIBS} ${GTEST_LIBRARIES} ${GTEST_MAIN_LIBRARIES} Threads::Threads)
+target_link_libraries(runISISTests isis3 ${isis_libs} ${ALLLIBS} ${GTEST_LIBRARIES} ${GTEST_MAIN_LIBRARIES} Threads::Threads)
```

Comment:
I'm assuming this created no symbol redefinition errors? If so we should probably ship everything in one lib. Nothing to do here just a thought.

Reply:
I didn't see any errors. I'll double-check.

Reply:
This shouldn't cause problems because each mission library links against the main library.